test/tselect.c

```diff
@@ -14236,7 +14236,7 @@ test_hyper_regular(void)
 **
 ****************************************************************/
 static void
-test_hyper_unlim_check(hid_t sid, hsize_t *dims, hssize_t enpoints, hssize_t enblocks, hsize_t *eblock1,
+test_hyper_unlim_check(hid_t sid, hsize_t *dims, hssize_t endpoints, hssize_t enblocks, hsize_t *eblock1,
```

Comment:
I think enpoints is correct (see enblocks, eblock1, eblock2) npoints is prefixed by 'e'